hawkular/metrics.py

```diff
@@ -124,7 +127,13 @@ def _clean_metric_id(metric_id):
         return quote(metric_id, '')
 
     def _get_base_url(self):
-        return "{0}://{1}:{2}/{3}/".format(self.scheme, self.host, str(self.port), self.path)
+        scheme = self.scheme.lower()
```

Comment:
I do not know if it's a bug in urllib2 or the way it should be, on my system:
```
urlopen('https://example.com/foo') - works
urlopen('https://example.com:443/foo') - not working

```
If it's just on my system, I will remove this ( and have a locale version with it for me )
it it's on all systems with python2 and urllib2, I think it should stay.

Can anyone verify that it's just on my system or something wider ?